inception-kb/src/test/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImplIntegrationTest.java

```diff
@@ -1240,8 +1240,8 @@ public void getConceptRoots_WithWildlifeOntology_ShouldReturnRootConcepts() thro
                 .map(KBHandle::getName);
 
         String[] expectedLabels = {
-            "Adaptation", "Animal Intelligence", "Collection", "Conservation Status", "Ecozone",
-            "Habitat", "Red List Status", "Taxon Name", "Taxonomic Rank"
+            "Adaptation", "AnimalIntelligence", "Collection", "ConservationStatus", "Ecozone",
```

Comment:
Why were the spaces removed from the expected labels?

Reply:
Without language it takes the name from the IRI , but with language it supports the label.